frontend/src/components/dialogs/SecretDialogDelete.vue

```diff
@@ -36,12 +36,8 @@ limitations under the License.
 
       <v-card-text>
         <v-container fluid>
-          <template v-slot:message>
-            <div>
-              Are you sure to delete the secret <span class="font-weight-bold">{{name}}</span>?<br/>
-              <span class="red--text font-weight-bold">The operation can not be undone.</span>
-            </div>
-          </template>
+          Are you sure to delete the secret <span class="font-weight-bold">{{name}}</span>?<br/>
```

Comment:
what happened to the `<template v-slot:message>`? It's not needed anymore?

Reply:
I asked myself the same question. I don't know but the slot doesn't exist anymore. This was the reason why the text did not show up. I also deleted the div as it is not required, but the template slot thingy was the reason for the issue